packages/rest/test/routes.test.ts

```diff
@@ -794,9 +796,9 @@ describe("rest: routes", () => {
         }
       `);
       strictEqual(ops[0].verb, "get");
-      strictEqual(ops[0].path, "/{id}");
+      strictEqual(ops[0].path, "/pets/{id}");
```

Comment:
My changes actually exposed a bug in the behavior being validated for this test; the `pets` segment was missing from the asserts!